webapp/channels/src/components/actions_menu/__snapshots__/actions_menu.test.tsx.snap

```diff
@@ -137,8 +137,8 @@ exports[`components/actions_menu/ActionsMenu has actions - marketplace enabled a
     />
     <MenuItemAction
       icon={
-        <span
-          className="icon-view-grid-plus-outline MenuItem__compass-icon"
+        <ActionsMenuIcon
+          name="icon-view-grid-plus-outline"
         />
```

Comment:
for consistency the component can be on a single line 
```suggestion
        <ActionsMenuIcon name="icon-view-grid-plus-outline"/>
```

Reply:
These `.snap` files are actually automatically generated by Jest, so it'd be better to just leave them as-is. Facebook makes Jest, so it'd likely complain if we touch this file at all 😛